readers/hdfs/src/main/java/com/criteo/hadoop/garmadon/hdfs/writer/ProtoParquetWriterWithOffset.java

```diff
@@ -196,10 +197,9 @@ private double getLatestCommittedTimestamp() {
         //there are cases for which we won't find a value for the latest committed timestamp
         // - the first time this code goes in, no file has the correct metadata
         // - for a new event type, we have no history too, so no value
-        //By using the default value 'now' rather than 0, we prevent firing unnecessary alerts
-        //However, if there is an actual problem and the reader never commits, it will eventually fire
-        //an alert.
-        long defaultValue = System.currentTimeMillis();
+        // - if started and no file has been create for the day
+        //Using the timestamp of today midnight prevents unnecessary alerts
+        long defaultValue = LocalDateTime.now().toLocalDate().atStartOfDay().toEpochSecond(ZoneOffset.UTC) * 1000;
```

Comment:
Why we need a changing name. maybe just make sure that alerting do not use the file name as source of information. and use a constant name that at least will be clear that file is created because there is nothing to commit

Reply:
Not sure where you see a file name. This is the timestamp for the monitoring/alerting part, it's not used for files naming (cf tests).